libraries/bot-applicationinsights/src/main/java/com/microsoft/bot/applicationinsights/core/TelemetryInitializerMiddleware.java

```diff
@@ -36,22 +36,20 @@ public TelemetryInitializerMiddleware(TelemetryLoggerMiddleware withTelemetryLog
                 this.httpContext = HttpClientContext.create();
             }
 
-            Object item = httpContext.getAttribute(TelemetryBotIdInitializer.BotActivityKey);
+            Object item = httpContext.getAttribute("BotBuilderActivity");
```

Comment:
could this have been a constant?

Reply:
We could also create the class `TelemetryBotIdInitializer` and only add the constant `BotActivityKey`

Reply:
I'd prefer to avoid creating the class. It's better to add the constant here

Reply:
Added